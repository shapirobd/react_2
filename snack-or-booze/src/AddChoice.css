.card-text {
	margin: 20px 0 10px 0 !important;
	text-align: center;
}

.card-text a {
	background-color: white;
	margin: 10px;
	padding: 5px 10px;
	border-radius: 0.5rem;
	color: black;
	text-decoration: none;
}

.card-text a:hover {
	background-color: gray;
	color: white;
}
