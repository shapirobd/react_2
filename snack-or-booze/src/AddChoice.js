import React from "react";
import { Link } from "react-router-dom";
import { Card, CardBody, CardTitle, CardText } from "reactstrap";
import "./AddChoice.css";

// COMPONENT - contains option to choose whether you'd like to add a snack or a drink, and then takes you to new form to add one once you click an option.
const AddChoice = () => {
	return (
		<section className="col-md-4">
			<Card>
				<CardBody>
					<CardTitle className="font-weight-bold text-center">
						Would you like to add a snack or a drink?
					</CardTitle>
					<CardText>
						<Link to="/add/snack">Snack</Link>
						<Link to="/add/drink">Drink</Link>
					</CardText>
				</CardBody>
			</Card>
		</section>
	);
};

export default AddChoice;
