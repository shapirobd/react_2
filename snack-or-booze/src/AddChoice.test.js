import React from "react";
import { MemoryRouter } from "react-router-dom";
import { render } from "@testing-library/react";
import AddChoice from "./AddChoice";

test("it should render without crashing", () => {
	render(
		<MemoryRouter>
			<AddChoice />
		</MemoryRouter>
	);
});

test("snapshot test", () => {
	const { asFragment } = render(
		<MemoryRouter>
			<AddChoice />
		</MemoryRouter>
	);
	expect(asFragment()).toMatchSnapshot();
});
