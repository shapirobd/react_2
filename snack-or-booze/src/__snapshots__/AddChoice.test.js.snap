// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<DocumentFragment>
  <section
    class="col-md-4"
  >
    <div
      class="card"
    >
      <div
        class="card-body"
      >
        <h5
          class="font-weight-bold text-center card-title"
        >
          Would you like to add a snack or a drink?
        </h5>
        <p
          class="card-text"
        >
          <a
            href="/add/snack"
          >
            Snack
          </a>
          <a
            href="/add/drink"
          >
            Drink
          </a>
        </p>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
